hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -1455,22 +1455,22 @@ public String getName() {
     @Override
     public void clear() {
         records.clear();
-        onClear();
     }
 
-    protected void onClear() {
+    public void clearAndSendInvalidation() {
```

Comment:
I was reluctant to make change to the interface, but as it is not in SPI and inside impl package I will do this.